frontend/src/assets/example_document.js

```diff
@@ -1,37 +1,37 @@
 import { getMarkForCommentThreadID } from "@/util/editorCommentUtils";
+import { getMarkForRedactionID } from "@/util/editorRedactionUtils";
 import { comments } from "./example_comments";
 
 const overlappingCommentThreadID = comments[0].id;
+import { v4 as uuid } from "uuid";
 
 const ExampleDocument = [
     {
         type: "paragraph",
         children: [
+            {
+                text:
+                    "You're invalid, because yes. ",
+            },
             {
                 text: "Seedlings Life Science Ventures, LLC [Seedlings]",
-                [getMarkForCommentThreadID(comments[1].id)]: true,
-                suggested: true,
+                [getMarkForRedactionID(uuid())]: true,
+                
             },
             {
                 text: " is in the business of early-stage health-care related research and product development. It alleges that Pfizer Canada ULC [Pfizer]",
-                [getMarkForCommentThreadID(overlappingCommentThreadID)]: true,
+                [getMarkForCommentThreadID(comments[1].id)]: true,
             },
             {
                 text: ", a major pharmaceutical company, infringes its patent by selling in Canada an auto-injector commonly known as the EpiPen. While, at first sight, the EpiPen and Seedlings’s invention do",
-                [getMarkForCommentThreadID(overlappingCommentThreadID)]: true,
```

Comment:
just curious - why did you remove the slate fields here? Intentional?

Reply:
This was for testing, will bring it back